Documentation/ApiOverview/LinkHandling/Index.rst

```diff
@@ -0,0 +1,70 @@
+.. include:: /Includes.rst.txt
+.. index:: Links
+.. _LinkHandling:
+
+=============
+Link handling
+=============
+
+Link entered in the backend in TYPO3 are stored in an internal format in the
+database.
+
+..  include:: /Images/ManualScreenshots/Backend/BackendLinkFormats.txt
+
+For example a link to the page with uid 42 is stored in a backend input as
+`t3://page?uid=42` and in the rich-text editor (RTE) as
+:html:`<a href="t3://page?uid=1">test</a>`.
+
+..  todo:
+    Make a chapter that describes the extended formats containing titles,
+    targets, link classes, etc
+
+Such links must be converted before they are output as HTML in the frontend.
+For example in Fluid all input from the RTE should be output by the ViewHelper
+:ref:`t3viewhelper:typo3-fluid-format-html`:
+
+..  code-block:: html
+    :caption: EXT:my_extension/Resources/Private/Templates/MyTemplate.html
+
+    <f:format.html>{myContent.bodytext}</f:format.html>
+
+Links provided in backend fields like the :sql:`header_link` can be used as
+input in the ViewHelper :ref:`t3viewhelper:typo3-fluid-link-typolink`:
+
+..  code-block:: html
+    :caption: EXT:my_extension/Resources/Private/Templates/MyTemplate.html
+
+    <f:link.typolink parameter="{myContent.header_link}">
+      {myContent.header_link}
+    </f:link.typolink>
+
+In TypoScript RTE content can be converted by the function :ref:`t3tsref:parsefunc`,
+link fields can be converted into HTML by the function :ref:`t3tsref:typolink`.
+
+In PHP context links are usually stored in an array format. Each link type is
```

Comment:
```suggestion
In PHP, context links are usually stored in an array format. Each link type is
```

What are "context links". It is not described above.
How are they stored in an array syntax?

Reply:
I adjusted  your suggestion: configured or passed is maybe a better word. And it is the PHP context as opposed to typoscript or Fluid

Reply:
Ah, the comma makes the difference :-)
But I don't get the array format. Do we have a section where the format is described so we can link it?

Reply:
That would be the todo bellow